packages/feedback-widget/src/fields.tsx

```diff
@@ -126,3 +126,94 @@ export const CategoryField: React.FC<React.PropsWithChildren<any>> = (
     </div>
   )
 }
+
+export const OptionalTextField: React.FC<React.PropsWithChildren<any>> = ({
```

Comment:
created two new fields for the form

Reply:
I don't think we're using this package much, apps have usually their own isolated feedback widget (see TT).